x/ibc/02-client/keeper/grpc_query_test.go

```diff
@@ -147,7 +156,6 @@ func (suite *KeeperTestSuite) TestQueryClientStates() {
 				for i := range expClientStates {
 					suite.Require().Equal(expClientStates[i].ClientId, res.ClientStates[i].ClientId)
 					suite.Require().NotNil(res.ClientStates[i].ClientState)
-					expClientStates[i].ClientState.ClearCachedValue()
```

Comment:
I had to do this bc the Equal statement would fail if the expected response had the value cached. Using Proto equal would work too

Reply:
It passes now when using the testing package. Going to leave as is but we can add back if necessary